dan_layer/template_lib/Cargo.toml

```diff
@@ -15,6 +15,7 @@ tari_crypto = "0.16"
 
 newtype-ops = "0.1.4"
 serde = { version = "1.0.143", default-features = false, features = ["derive", "alloc"] }
+serde_json = { version = "1.0.64" }
```

Comment:
The template lib must have minimal dependencies as they will be compiled into every WASM.